webapp/channels/src/components/widgets/menu/menu_items/menu_item.scss

```diff
@@ -327,16 +326,37 @@
             color: var(--link-color);
             font-style: normal;
             font-weight: 600;
-        }
 
-        button:hover {
-            background-color: transparent;
+            &:hover {
+                background: transparent;
+                text-decoration: underline;
+            }
         }
 
         .start_trial_content {
             position: relative;
             width: 100%;
         }
+
+        .free_version_badge {
```

Comment:
Maybe this new CSS can be in a styled component?

Reply:
Moved to styled component.